guides/bulk.md

```diff
@@ -0,0 +1,271 @@
+# Bulk
+
+In this guide, you'll learn how to use the OpenSearch Rust Client API to perform bulk operations. You'll learn how to index, update, and delete multiple documents in a single request.
+
+## Setup
+
+First, create a client instance with the following code:
+
+```rust
+let url = Url::parse("https://localhost:9200")?;
+let credentials = Credentials::Basic("admin".into(), "admin".into());
+let transport = TransportBuilder::new(SingleNodeConnectionPool::new(url))
+    .cert_validation(CertificateValidation::None)
+    .auth(credentials)
+    .build()?;
+let client = OpenSearch::new(transport);
+```
+
+Next, create an index named `movies` and another named `books` with the default settings:
+
+```rust
+let movies = "movies";
+let books = "books";
+client.indices().create(IndicesCreateParts::Index(movies)).send().await?;
+client.indices().create(IndicesCreateParts::Index(books)).send().await?;
+```
+
+## Bulk API
+
+The `bulk` API action allows you to perform document operations in a single request. The body of the request is an array of objects that contains the bulk operations and the target documents to index, create, update, or delete.
+
+### Indexing multiple documents
```

Comment:
Due to the semantics of "create" generally assuming an id is being specified, I believe it makes sense to swap the operation types of this section with the next one.
```suggestion
### Creating multiple documents
```